app/src/main/java/com/summer/practice/tvtracker/networking/Api.kt

```diff
@@ -0,0 +1,22 @@
+package com.summer.practice.tvtracker.networking
+
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+const val baseUrl = "https://api.themoviedb.org/3/"
+const val apiKey = "0a416fc6c49f4a04db6e3bd398ef8579"
```

Comment:
if you want something challenging, you can get rid of the apiKey param in those GET methods and add it to retrofit interceptor